index.html

```diff
@@ -34,6 +34,7 @@
 		<div class="wrapper">
 			<!-- This box contains the large image and text first seen on page -->
 			<div class="box1">
+				<img src="images/index_large.jpg">

```

Comment:
Included an image for the Index page which is the first thing users will see when they visit my website.